react/src/Calendar/Calendar.tsx

```diff
@@ -34,6 +34,8 @@ export interface CalendarProps extends CalendarBaseProps {
   isDateUnavailable?: (d: Date) => boolean
   /** Color scheme for component */
   colorScheme?: ThemingProps<'Calendar'>['colorScheme']
+  /** The date to focus when calendar first renders. */
+  defaultFocusedDate?: Date
```

Comment:
you may want this to be in `CalendarBaseProps` instead, so `RangeCalendar` (and by extension `DatePicker`/`DateRangePicker`) also gets the props instead of redundant declaration